src/app/products/fixed-deposit-products/create-fixed-deposit-product/create-fixed-deposit-product.component.ts

```diff
@@ -10,6 +10,7 @@ import { FixedDepositProductChargesStepComponent } from '../fixed-deposit-produc
 import { FixedDepositProductAccountingStepComponent } from '../fixed-deposit-product-stepper/fixed-deposit-product-accounting-step/fixed-deposit-product-accounting-step.component';
```

Comment:
Shall we add headers(comments) for the imports?  You can get an idea by looking at edit-charge.component.ts file